Test.Utility/TestFile.cs

```diff
@@ -1,21 +1,27 @@
-﻿namespace Messerli.Test.Utility
+﻿using System;
+
+namespace Messerli.Test.Utility
 {
     public class TestFile
     {
         public string SourceFilePath { get; }
 
         public string RelativeFilePath { get; }
 
-        public TestFile(string filePath)
-        {
-            SourceFilePath = filePath;
-            RelativeFilePath = filePath;
-        }
-
         public TestFile(string sourceFilePath, string relativeFilePath)
         {
             SourceFilePath = sourceFilePath;
             RelativeFilePath = relativeFilePath;
         }
+
+        public static TestFile Create(string filePath)
+        {
+            return new TestFile(filePath, filePath);
+        }
+
+        public static TestFile Create(Type type, string relativeFilePath)

```

Comment:
Pass the `Assembly` directly

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/13/commits/49c6b592df5a4461e314d6690769b9f85f7aad66